Client/Frontend/Settings/WebsiteRedirectsSettingsView.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+
+struct WebsiteRedirectsSettingsView: View {
+  @ObservedObject private var reddit = Preferences.WebsiteRedirects.reddit
+  @ObservedObject private var npr = Preferences.WebsiteRedirects.npr
+  
+  var body: some View {
+    List {
+      Section(footer: Text(Strings.redditRedirectFooter)) {
+        // Cell titles for websites do not have to be translated.
+        Toggle(isOn: $reddit.value) {
+          Text("reddit.com \(Image(systemName: "arrow.right")) old.reddit.com")
+        }
+        .toggleStyle(SwitchToggleStyle(tint: .accentColor))
+      }
+      .listRowBackground(Color(.secondaryBraveGroupedBackground))
```

Comment:
As of iOS 16, having the `listRowBackground` like this will lead to a bug where the header and footer view's will also get the background.  Best to place this on a `Group` inside the Section body or since this is only 1 view, just place it on that view (the Toggle in this case) instead